lib/matplotlib/dviread.py

```diff
@@ -1079,10 +1082,7 @@ def find_tex_file(filename, format=None):
         result = cbook._check_and_log_subprocess(cmd, _log, **kwargs)
     except RuntimeError:
         return ''
-    if os.name == 'nt':
-        return result.decode('utf-8').rstrip('\r\n')
```

Comment:
Why don't we have to strip the `\r` anymore?

Reply:
because passing an `encoding` also turns on newline conversion, per https://docs.python.org/3/library/subprocess.html#frequently-used-arguments.